RELEASE_NOTES.md

```diff
@@ -75,9 +78,10 @@ Any new bugs, requests, or missing features should be reported as [Github issues
  * zcbor.py: Fix error in 'convert' where a bstr dict value was incorrectly translated
  * setup.py: Fix windows install
  * zcbor.py: Add suffix for literal values larger than 32-bits.
- * zcbor.py: Make changes to generated code to build with `-Wpedantic` and `-Wconversion`.
  * src: Fix code to build with `-Wpedantic` and `-Wconversion`.
  * zcbor.py: Fix some code generation bugs to do with range checking (min/max size) of ints and CBOR-encoded bstrs.
+ * zcbor.py: Fix bug where indirection for ints with range checks was wrong.
+ * zcbor.py: Fix bugs in range checks on .cbor bstrs.
```

Comment:
> on .cbor bstrs.

nit: this looks odd to me with the `.cbor`. Not sure if this needs inline syntax or if this referencing something specific that I'm not getting, but feel free to ignore if you feel this is the best way to write it.

Reply:
Changed to "Fix bugs in range checks in `bstr`s with `.cbor` statements."